mathesar_ui/src/systems/record-view/TableWidget.svelte

```diff
@@ -49,12 +51,30 @@
   $: tabularDataStore.set(tabularData);
   $: ({ currentRolePrivileges } = table.currentAccess);
   $: canViewTable = $currentRolePrivileges.has('SELECT');
+  $: getTablePageUrl = $storeToGetTablePageUrl;
+  $: href = getTablePageUrl
+    ? getTablePageUrl({ tableId: table.oid })
+    : undefined;
```

Comment:
`getTablePageUrl` is always defined. This check is unnecessary.

```suggestion
  $: href = getTablePageUrl({ tableId: table.oid });
```